src/DssSpell.t.sol

```diff
@@ -473,20 +473,37 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(arbitrumGateway.validDomains(arbDstDomain), 0, "l2-arbitrum-invalid-dst-domain");
     }
 
-    // For PE-1217
+    struct VaultParams {
+        uint256 line;
+        uint256 debt;
+    }
+
+    // Avoids issues with stack too deep in `testGlobalDebtCeilingDecrease`
+    function _getDebtCeilingParams(bytes32 ilk) internal view returns (VaultParams memory) {
+        (uint256 Art, uint256 rate, , uint256 line, ) = vat.ilks(ilk);
+
+        return VaultParams({
+            line: line,
+            debt: Art * rate
+        });
+    }
 
-    function testNewModulesAuthorizingEsm() public {
-        uint256 ward;
-        address ESM = addr.addr("MCD_ESM");
+    function testGlobalLineDecrease() public {
+        VaultParams memory rwa008A = _getDebtCeilingParams("RWA008-A");
+        VaultParams memory yfiA    = _getDebtCeilingParams("YFI-A");
+        VaultParams memory maticA  = _getDebtCeilingParams("MATIC-A");
+        VaultParams memory linkA   = _getDebtCeilingParams("LINK-A");
+        uint256 sumLines         = rwa008A.line + linkA.line + maticA.line + yfiA.line;
+        uint256 sumDebts         = rwa008A.debt + linkA.debt + yfiA.debt + maticA.debt;
+        uint256 lineReduction    = 90 * (sumLines > sumDebts ? sumLines - sumDebts : 0) / 100;
 
-        ward = WardsAbstract(addr.addr("MIP21_LIQUIDATION_ORACLE")).wards(ESM);
-        assertEq(ward, 0, "unexpected ward");
+        uint256 globalLineBefore = vat.Line();
 
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        ward = WardsAbstract(addr.addr("MIP21_LIQUIDATION_ORACLE")).wards(ESM);
-        assertEq(ward, 1, "MIP21_LIQUIDATION_ORACLE does not authorize ESM");
+        uint256 globalDcAfter = vat.Line();
+        assertEq(globalDcAfter, globalLineBefore - lineReduction, "TestError/global-dc-not-updated");
```

Comment:
It literally doesn't matter because of the commutative property of addition.
Also, I don't think someone looking at this in the future would do it looking at this week spreadsheet, so it wouldn't even be a matter of "setting precedence".

Still, I'll make this change.